runtime/jnichk/jnicwrappers.c

```diff
@@ -258,13 +258,14 @@ static jobjectRefType JNICALL checkGetObjectRefType(JNIEnv *env, jobject obj);
 static jint JNICALL
 checkGetVersion(JNIEnv *env)
 { 	
-	J9JavaVM* j9vm = ((J9VMThread*)env)->javaVM;
+	J9JavaVM* j9vm = NULL;
```

Comment:
Again, just use the global in all these cases.

Reply:
I updated all the usages of the vm in jnicwrappers.c, but I wasn't sure if you thought all the usages in jnicheck.c (outside of jniCheckArgs.c) should also be updated?

Reply:
Ideally, all uses should be updated (all uses have to at least be examined to prove the initial verification has been done).